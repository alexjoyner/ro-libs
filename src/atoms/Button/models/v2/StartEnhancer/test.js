import React from 'react';
import { shallow } from 'enzyme';
import { StartEnhancer } from '.';

describe('StartEnhancer component', () => {
  let component;
  beforeEach(() => {
    component = shallow(<StartEnhancer />);
  });
  it('Should render without exploding', () => {
    expect(component).toBeDefined();
  });
});
