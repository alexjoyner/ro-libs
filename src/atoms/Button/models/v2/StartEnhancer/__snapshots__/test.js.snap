// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartEnhancer component Should match snapshot 1`] = `undefined`;
